featuretools/tests/conftest.py

```diff
@@ -746,6 +748,35 @@ def rolling_outlier_series_pd():
     )
 
 
+@pytest.fixture
```

Comment:
Do we need these as all separate fixtures, if we are only using the dataframe fixture?

Reply:
Along those lines - we also might want to test with Dask and Spark inputs, if since we are saying these primitives are also compatible with those libraries.

Also, does Woodwork force the type of the data once initialized? Is the data always stored as a string after Woodwork initialization? If so, there is no real need to test different input types (int vs string) here as we are just essentially testing that Woodwork is converting as we expect upon initialization, and that should be handled in the WW tests.

Reply:
I think you are right that in the test cases here, the dtype is always a string categorical. 

However, I found this edge case where the dtype can be a float categorical: 

``` 
ser = pd.Series([12345, 67890]).astype('category')
ser = ww.init_series(ser, logical_type='PostalCode') 
```

In this case it appears in my jupyter notebook the type each element is a float.
 
I can update the tests here to reflect that case. I suspect this is related to the fact that there are multiple types of data that can hold the categorical dtype in pandas. If I remove the call to `astype`, the dtype is coerced to a string categorical. Woodwork seems to only check that the dtype is categorical, but not which type of categorical.

Reply:
I think this might be a Woodwork issue. I think we should always expect the data to be of the same type once Woodwork is initialized. Let's file an issue in WW for this edge case. If we are coercing other types to string, I think we should do the same with a categorical input as well.

Reply:
Done! I've also updated the tests to test this particular edge case for now.